docs/react-testing-library/example-intro.md

```diff
@@ -127,7 +127,7 @@ expect(getByTestId('greeting-text')).toHaveTextContent('hello there')
 expect(getByTestId('ok-button')).toHaveAttribute('disabled')
 
 // snapshots work great with regular DOM nodes!
-expect(container.firstChild).toMatchSnapshot()
+expect(container).toMatchSnapshot()
```

Comment:
Actually, while we're at it, maybe we should also swap `.toMatchSnapshot` with `.toMatchInlineSnapshot`. This will help to encourage people to keep snapshots smaller. This will require that we figure out what the snapshot will really look like so we can inline it. You up for that @Agentkma? If you need help let us know.

Reply:
Yes, I'm up for it. I will look into and let you know if I need help.  I've just started learning and using this library, so this is a great opportunity to learn while helping improve things.  Win-Win in my book :)

Reply:
Ok, so would we want to show a code snippet/or full component of the <Fetch/> component(like the Svelt Testing example does?) to show how we find what the string arg to .toMatchInlineSnapshot() ? 
![Screen Shot 2019-08-01 at 8 18 54 AM](https://user-images.githubusercontent.com/22228809/62301109-32334d00-b435-11e9-9fea-a1c9c4c45fec.png)

Reply:
Yeah, that sounds good. It'll probably need to be something like:

```javascript
expect(container).toMatchInlineSnapshot(`
  <div>
    <h1>
      Hello
      {name}
      !
    </h1>
  </div>
`)
```

I'm not 100% certain that's how it will be serialized, but I'm pretty sure that's what it would look like.